extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -44,14 +45,85 @@ require([
 		});
 	}
 
+	function processData(threads, baseUrl, upvoteUrl) {
+		var ret = [],
+			i,
+			thread,
+			userData;
+
+		for (i in threads) {
+			thread = threads[i];
+			userData = thread._embedded.userData[0];
+
+			ret.push({
+				author: thread.createdBy.name,
+				authorAvatar: thread.createdBy.avatarUrl,
+				commentCount: thread.postCount,
+				content: thread.rawContent,
+				// TODO: format createdAt to string
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)